packages/test/test-end-to-end-tests/src/test/flattenRuntimeOptions.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IContainerRuntimeOptions } from "@fluidframework/container-runtime";
+
+/** Temporary function to make backwards compatible runtime options. */
+export function flattenRuntimeOptions(
```

Comment:
This is to use when tests run against older container runtime?

Reply:
Right